services/order.py

```diff
@@ -0,0 +1,34 @@
+from db.models import Ticket, Order, User, MovieSession
+from datetime import datetime
+
+from django.db import transaction
+from django.db.models import QuerySet
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> None:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = date
+            order.save()
+        for ticket in tickets:
+            movie_session = MovieSession.objects.get(
+                id=ticket["movie_session"]
+            )
```

Comment:
Why do you need fetch object, when you can use id in .create movie_session_id=id of movie_session